### PR TITLE
Testing branch

### PR DESCRIPTION
**Pull request by ahmedtaa**
# In this Branch :
> implemented tests for Add and remove tasks

### CODE REVIEW

File: src/Task.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// describe for delete;
+// describe for create;
```
Comment (julie-ify) [inline src/Task.test.js]: - [ ] Let's keep out code clean by removing commented codes

File: src/tasks_manager.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// import TasksManager from './tasks_manager';
```
Comment (julie-ify) [inline src/tasks_manager.test.js]: - [ ] Let's keep out code clean by removing commented codes

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi team,

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- All tests are passing ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @ahmedtaa and @zieeco,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**